catalog/ui/catalog-ui-search/src/main/webapp/component/list-editor/list-editor.view.js

```diff
@@ -0,0 +1,155 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+/*global define, require, module*/
+var Marionette = require('marionette');
+var _ = require('underscore');
+var $ = require('jquery');
+var template = require('./list-editor.hbs');
+var CustomElements = require('js/CustomElements');
+require('behaviors/button.behavior');
```

Comment:
Should we be assigning this to a var?

Reply:
The behaviors are a bit strange, where the act of requiring them is what causes a side effect.  They're like backbone associations (they don't get assigned) in that they're mixed into the Marionette framework.  All this does is make sure behaviors have been mixed in so that the current view can use them.